### DISCUSSION TITLE
Update?

### DISCUSSION THREAD

**Discussion by DataZombies** [yannbertrand/macos-defaults#287] [General]
Please update and test for Catalina, Big Sur, Monterey, Ventura and Sonoma.

**Comment by yannbertrand** [comment]
Thanks for reaching out and showing interest. I'm really passionate about this project and I work on it during my free time, juggling it with other commitments. I totally get the importance of updates, and I'm doing my best to make them happen. If you'd like to support the project and buy me a coffee or two, you can check out [my GitHub Sponsors page](https://github.com/sponsors/yannbertrand/). Your understanding and support mean a lot!
